backend/src/main/java/com/crm/dto/response/CarResponse.java

```diff
@@ -8,9 +8,10 @@
 @Builder
 @EqualsAndHashCode
 @NoArgsConstructor
-@AllArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
W tej klasie chyba też

Reply:
To nie zadziała z Builderem, zostawiłem jak było